TWLight/urls.py

```diff
@@ -88,6 +88,11 @@
     url(r"^contact/$", ContactUsView.as_view(), name="contact"),
     url(r"^$", NewHomePageView.as_view(), name="homepage"),
     url(r"^about/$", TemplateView.as_view(template_name="about.html"), name="about"),
+    url(
+        r"^search$",
```

Comment:
Did you leave the `/` out on purpose? Otherwise, I think you should add it

Reply:
Yeah, it really is out of alignment with literally every other route. Fixing!

Reply:
Done!